apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -52,6 +65,15 @@ const App: React.FC = () => {
 			<Loader isLoading={dataStatus === DataStatus.PENDING} withOverlay />
 
 			<img alt="logo" className="App-logo" src={reactLogo} width="30" />
+			{isRoot && (
+				<div className={styles["start-meeting"]}>
+					<Button label="Start a meeting" onClick={handleOpenModal} />
+				</div>
+			)}
+
+			<Modal isOpen={isModalOpen} onClose={handleCloseModal}>
+				<MeetingForm onClose={handleCloseModal} />
+			</Modal>
```

Comment:
This logic shouldn't be in app.tsx. Create a separate pages component in pages for root page